include/RED4ext/Handle.hpp

```diff
@@ -194,9 +194,11 @@ class Handle : public HandleBase
     }
 
 protected:
+    friend class WeakHandle<T>;
```

Comment:
This shouldn't be a friend class. You can use `HandlerBase` for this or forward declare it.